.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -0,0 +1,53 @@
+## ARM API Information (Control Plane)
```

Comment:
It might not be easy for user to select template. Should we put this `control_plane_template.md` back to `PULL_REQUEST_TEMPLATE.md` as default (as there is still far more ARM PR than data-plane PR), and ask data-plane PR to choose the `data_plane_template` template via query parameter?